easybuild/easyconfigs/p/Perl/Perl-5.26.0-GCCcore-6.4.0.eb

```diff
@@ -537,12 +537,10 @@ exts_list = [
         'checksums': ['b4f3f63b80f680a421332d8851ddbe5a8e72fcaa74d5d1d98f3c8cc4a3ece293'],
     }),
     ('XML::Bare', '0.53', {
-        'patches': ['XML-Bare-0.53_icc.patch'],
         'source_tmpl': 'XML-Bare-%(version)s.tar.gz',
         'source_urls': ['https://cpan.metacpan.org/authors/id/C/CO/CODECHILD'],
         'checksums': [
             '865e198e98d904be1683ef5a53a4948f02dabdacde59fc554a082ffbcc5baefd',  # XML-Bare-0.53.tar.gz
```

Comment:
please collapse this to a single line to adher to style used for other easyconfigs (likewise in other easyconfig) 

```python
'checksums': ['865e198e98d904be1683ef5a53a4948f02dabdacde59fc554a082ffbcc5baefd'],